core/src/Streamly/Internal/Data/Array/Unboxed.hs

```diff
@@ -103,6 +108,10 @@ module Streamly.Internal.Data.Array.Unboxed
     -- ** Folding
     , streamFold
     , fold
+
+    -- * Deprecated
+    , A.toStream
+    , A.toStreamRev
```

Comment:
Instead, would you like to move these to the released module? We can remove the deprecated header here and we will track deprecations more nicely.